examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.wfanet.examples.streaming
+
+import com.google.common.truth.Truth.assertThat
+import io.grpc.inprocess.InProcessChannelBuilder
+import io.grpc.inprocess.InProcessServerBuilder
+import io.grpc.testing.GrpcCleanupRule
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class HelloStreamingTest {
+  @Rule
+  @JvmField
+  val grpcCleanup = GrpcCleanupRule()
+
+  lateinit var client: HelloStreamingClient
+
+  @Before
+  fun setup() {
+    val serverName = InProcessServerBuilder.generateName()
+    grpcCleanup.register(
+      InProcessServerBuilder.forName(serverName)
+        .directExecutor()
+        .addService(HelloStreamingServer.HelloStreamingService())
+        .build()
+        .start()
+    )
+    val channel = InProcessChannelBuilder.forName(
+      serverName
+    ).directExecutor().build()
+    client = HelloStreamingClient(grpcCleanup.register(channel))
+  }
+
+  @Test
+  fun testStuff() = runBlocking {
```

Comment:
Better name? Use Kotlin standard of multiple words in backticks.

Reply:
Done